src/renderer/components/+namespaces/namespace.store.ts

```diff
@@ -97,13 +97,16 @@ export class NamespaceStore extends KubeObjectStore<Namespace> {
     return this.selectedNamespaces;
   }
 
-  getSubscribeApis() {
-    // if user has given static list of namespaces let's not start watches because watch adds stuff that's not wanted
+  subscribe() {
```

Comment:
Why did we change this?

Reply:
Removing `getSubscribeApis()` very much simplifies the `subscribe()` logic because it is not fixed at always 1 API.